CMakeLists.txt

```diff
@@ -683,14 +683,6 @@ if (NOT vast_is_subproject AND NOT TARGET configure)
   add_custom_target(configure COMMAND ${CMAKE_CURRENT_BINARY_DIR}/config.status)
 endif ()
 
-set(EXECUTABLE_OUTPUT_PATH
-    ${CMAKE_CURRENT_BINARY_DIR}/bin
-    CACHE PATH "Single directory for all executables")
-
-set(LIBRARY_OUTPUT_PATH
-    ${CMAKE_CURRENT_BINARY_DIR}/lib
-    CACHE PATH "Single directory for all libraries")
-
```

Comment:
Removing these lines means the files go somewhere else, but the multi-config generators still work correctly if they are kept. The `$<TARGET_FILE:>` changes are sufficient.

Reply:
I've added them back.